rollup.config.js

```diff
@@ -2,25 +2,68 @@ import { terser } from 'rollup-plugin-terser'
 import pkg from './package.json'
 import { nodeResolve } from '@rollup/plugin-node-resolve'
 import commonjs from '@rollup/plugin-commonjs'
+import { babel } from '@rollup/plugin-babel'
 
-const umd = { format: 'umd', name: 'A11yDialog', exports: 'default' }
-const es = { format: 'es' }
-const minify = {
-  plugins: [terser()],
-  banner: () => `/*! a11y-dialog ${pkg.version} — © Kitty Giraudel */`,
+const babelCfg = {
+  babelHelpers: 'bundled',
+  assumptions: {
+    enumerableModuleMeta: true,
+    mutableTemplateObject: true,
+    noClassCalls: true,
+    noDocumentAll: true,
+  },
+  presets: [['@babel/preset-env', { targets: 'ie 11' }]],
```

Comment:
Is it really needed to transpile to IE11? As IE11 is out of support, when opening IE11 it suggets to download edge. @KittyGiraudel @mxmason

Reply:
Downleveling es6 to es5 makes the non-esm build about 0.2kb bigger than it used to be, and 0.3kb bigger than the ESM build that this branch produces.

ESM: 1.2kb (currently 1.3kb in prod)
UMD: 1.5kb (currently 1.3kb in prod)

I'm comfortable with shipping the additional kb to maintain support if you are. It's no real cost to our developer experience, as far as I can see it.

Reply:
Yep, let’s keep IE11 support. That’s marginal if anything.